src/cascade/executor/dismodel_main.py

```diff
@@ -0,0 +1,103 @@
+"""
+Entry point for running a the work of a single location in an EpiViz-AT cascade.
+"""
+import os
+from bdb import BdbQuit
+from datetime import timedelta
+from pathlib import Path
+from pprint import pformat
+from timeit import default_timer
+
+from pkg_resources import get_distribution, DistributionNotFound
+
+from cascade.core import getLoggers
+from cascade.executor.argument_parser import DMArgumentParser
+from cascade.executor.cascade_plan import CascadePlan
+from cascade.input_data.configuration import SettingsError
+from cascade.input_data.db.configuration import load_settings
+from cascade.input_data.db.locations import location_hierarchy
+from cascade.testing_utilities import make_execution_context
+from cascade.executor.estimate_location import estimate_location
+
+CODELOG, MATHLOG = getLoggers(__name__)
+
+
+def main(args):
+    start_time = default_timer()
+    execution_context = make_execution_context(gbd_round_id=5)
+    settings = load_settings(execution_context, args.meid, args.mvid, args.settings_file)
+    locations = location_hierarchy(execution_context)
+    plan = CascadePlan.from_epiviz_configuration(locations, settings)
+
+    for cascade_task_identifier in plan.cascade_jobs:
+        cascade_job, this_location_work = plan.cascade_job(cascade_task_identifier)
+
+        if cascade_job == "bundle_setup":
+            pass  # Move bundle to next tier
+        elif cascade_job == "estimate_location":
+            estimate_location(execution_context, this_location_work)
+
+    elapsed_time = timedelta(seconds=default_timer() - start_time)
+    MATHLOG.debug(f"Completed successfully in {elapsed_time}")
+
+
+def entry(args=None):
+    """Allow passing args for testing."""
+    readable_by_all = 0o0002
+    os.umask(readable_by_all)
+
+    parser = DMArgumentParser("Run DismodAT from Epiviz")
+    parser.add_argument("db_file_path", type=Path)
+    parser.add_argument("--settings-file", type=Path)
+    parser.add_argument("--no-upload", action="store_true")
+    parser.add_argument("--db-only", action="store_true")
+    parser.add_argument("-b", "--bundle-file", type=Path)
+    parser.add_argument("-s", "--bundle-study-covariates-file", type=Path)
+    parser.add_argument("--skip-cache", action="store_true")
+    parser.add_argument("--num-processes", type=int, default=4,
+                        help="How many subprocesses to start.")
+    parser.add_argument("--pdb", action="store_true")
+    args = parser.parse_args(args)
+
+    try:
+        software_version = get_distribution("cascade").version
```

Comment:
Should we put this in `cascade.core.__version__`?

Reply:
Why in core? I'm not opposed. I haven't seen that pattern. I could see putting it into `cascade.__version__` b/c that's a common pattern, like for numpy.

Reply:
`cascade` is a namespace package, otherwise that is certainly where it would go

Reply:
You want it in core b/c this is a namespace package. I figured out the answer, and I put it there.